ui/src/application/UpdateApplicationDialog.tsx

```diff
@@ -0,0 +1,93 @@
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import TextField from '@material-ui/core/TextField';
+import Tooltip from '@material-ui/core/Tooltip';
+import React, {Component} from 'react';
+
+interface IProps {
+    fClose: VoidFunction;
+    fOnSubmit: (name: string, description: string) => void;
+    initialName: string;
+    initialDescription: string;
+}
+
+interface IState {
+    name: string;
+    description: string;
+}
+
+export default class UpdateDialog extends Component<IProps, IState> {
+    public state = {name: '', description: ''};
+
+    public componentWillMount() {
+        this.setState({name: this.props.initialName, description: this.props.initialDescription});
+    }
+
+    public render() {
+        const {fClose, fOnSubmit} = this.props;
+        const {name, description} = this.state;
+        const submitEnabled = this.state.name.length !== 0;
+        const submitAndClose = () => {
+            fOnSubmit(name, description);
+            fClose();
+        };
+        return (
+            <Dialog
+                open={true}
+                onClose={fClose}
+                aria-labelledby="form-dialog-title"
+                id="app-dialog">
+                <DialogTitle id="form-dialog-title">Update an application</DialogTitle>
+                <DialogContent>
+                    <DialogContentText>
+                        An application is allowed to send messages.
+                    </DialogContentText>
+                    <TextField
+                        autoFocus
+                        margin="dense"
+                        className="name"
+                        label="Name *"
+                        type="email"
```

Comment:
```suggestion
                        type="text"
```

Reply:
I think I copied it from an existing code:(. Maybe update them both?

https://github.com/gotify/server/blob/master/ui/src/application/AddApplicationDialog.tsx#L43-L49

Reply:
Yeah, please fix both (:.